### DISCUSSION TITLE
Negative prices

### DISCUSSION THREAD

**Discussion by ClausSamuelsen** [UtilitechAS/amsreader-firmware#619] [Show and tell]
Just want to share this rare price screenshot. Negative prices all day long :)
<img width="1526" alt="Negative prices 2023-08-08 at 08 16 09" src="https://github.com/UtilitechAS/amsreader-firmware/assets/96246629/36ad126e-1a62-49a1-8981-6c86c5a473e2">

**Comment by gskjold** [comment]
Enjoy it while it lasts!
